CMakeLists.txt

```diff
@@ -527,6 +527,22 @@ endif(BUILD_SPARSE)
 
 if(BUILD_GRAPHBOLT)
   message(STATUS "Configuring graphbolt library")
+  if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+    include(ExternalProject)
+    ExternalProject_Add(
+        liburing
+        SOURCE_DIR ${CMAKE_SOURCE_DIR}/third_party/liburing
+        CONFIGURE_COMMAND <SOURCE_DIR>/configure --prefix=<INSTALL_DIR>
+        BUILD_COMMAND bash -c "make -j `nproc`"
```

Comment:
compile with all cores may not be necessary and may hit host perf though liburing is unlikely to use all cores actually. How about divide with `4` or just fix with `-j 4` if building liburing is not heavy at all?